lib/documents/schemas/residential_property_tribunal_decisions.json

```diff
@@ -68,179 +68,179 @@
-          "value": "leasehold-enfranchisement-extension---houses-enfranchisement"
+          "label": "Leasehold enfranchisement and extension - Houses - enfranchisement",
+          "value": "leasehold-enfranchisement-and-extension---houses-enfranchisement"
         },
         {
-          "label": "Leasehold Enfranchisement & Extension - Flats - Collective Enfranchisement",
-          "value": "leasehold-enfranchisement-extension---flats-collective-enfranchisement"
+          "label": "Leasehold enfranchisement and extension - Flats - collective enfranchisement",
+          "value": "leasehold-enfranchisement-and-extension---flats-collective-enfranchisement"
         },
         {
-          "label": "Leasehold Enfranchisement & Extension - Right of First Refusal",
-          "value": "leasehold-enfranchisement-extension---right-of-first-refusal"
+          "label": "Leasehold enfranchisement and extension - Right of first refusal",
+          "value": "leasehold-enfranchisement-and-extension---right-of-first-refusal"
         },
         {
-          "label": "Park Homes - Pitch Fee",
+          "label": "Park homes - Pitch fee",
           "value": "park-homes---pitch-fee"
         },
         {
-          "label": "Park Homes - Section 4 - Any Question Under Act or Agreement",
+          "label": "Park homes - Section 4 - any Question Under Act or Agreement",
           "value": "park-homes---section-4-any-question-under-act-or-agreement"
         },
         {
-          "label": "Park Homes - Local Authority Compliance Notice",
+          "label": "Park homes - Local authority compliance notice",
           "value": "park-homes---local-authority-compliance-notice"
         },
         {
-          "label": "Park Homes - Appeal Against/Alteration of a Site Licence Condition",
+          "label": "Park homes - Appeal against/alteration of a site licence condition",
           "value": "park-homes---appeal-against-alteration-of-a-site-licence-condition"
         },
         {
-          "label": "Park Homes - Local Authority Decision - Emergency Action",
+          "label": "Park homes - Local authority decision - emergency action",
           "value": "park-homes---local-authority-decision-emergency-action"
         },
         {
-          "label": "Park Homes - Local Authority Site Licence Refusal/Refusal to Transfer",
+          "label": "Park homes - Local authority site licence refusal/refusal to transfer",
           "value": "park-homes---local-authority-site-licence-refusal-refusal-to-transfer"
         },
         {
-          "label": "Park Homes - Site Owner's Failure To Deposit Site Rules with LA",
-          "value": "park-homes---site-owners-failure-to-deposit-site-rules with la"
+          "label": "Park homes - Site owner's failure to deposit site rules with local authority",
+          "value": "park-homes---site-owners-failure-to-deposit-site-rules-with-la"
```

Comment:
Replace: `-la` at the end with: `-local-authority` so the value matches up with the label